REST/RESTListener+Replicate.cc

```diff
@@ -190,20 +204,26 @@ namespace litecore { namespace REST {
         }
 
 
-        HTTPStatus wait() {
-            Lock lock(_mutex);
-            _cv.wait(lock, [this]{return finished();});
-            return _finalResult;
-        }
-
-
         void stop() override {
             Lock lock(_mutex);
             if (_repl) {
                 c4log(ListenerLog, kC4LogInfo, "Replicator task #%u stopping...", taskID());
                 _repl->stop();
             }
         }
+        
+
+        void setStopOnIdle(bool stopOnIdle) {
```

Comment:
We do not need a "stop on idle" mode. That's identical to a one-shot replication.